achievements/labelBabyJunior.achievement.js

```diff
@@ -0,0 +1,22 @@
+var labelBabyJunior = {
+	name: 'Label Baby Junior',
+	check: function(pullRequest, shall) {
+		if (isManyLabels(pullRequest))  {
+			var achievement = {
+				avatar : 'images/achievements/theLabelMaker.achievment.jpg',
+				name: 'The Label Maker',
+				short: 'Is this a label maker?',
+				description: 'You\'ve put many labels, thank you for organizng. You\'re a gift that keeps on re-giving' ,
+				relatedPullRequest: pullRequest._id
+			};
+
+			shall.grant(pullRequest.creator.username, achievement);
+		}
+	}
+};
+
+function isManyLabels(pullRequest) {
+	var labels = pullRequest.labels;
+	return labels && labels.length > 3;
+
```

Comment:
missing a `}`

please check that your code is running before you `push` to Github :-)